test/data/endpoints.json

```diff
@@ -0,0 +1,18 @@
+{
+  "endpoints": {
+    "v1": {
+      "version": "1.1.2",
+      "signalk-http": "http://192.168.1.2/signalk/v1/api/",
+      "signalk-ws": "ws://192.168.1.2:34567/signalk/v1/stream",
+      "signalk-tcp": {
```

Comment:
I suppose `tcp://localhost:5555` won't cut it, even if this makes tcp the oddboll of the three.